src/es-check.js

```diff
@@ -2,57 +2,127 @@ const crypto = require('crypto');
 const path = require('path');
 const shell = require('shelljs');
 const promiseSpawn = require('./promise-spawn.js');
+const esCheck = require.resolve('es-check');
+const exitHook = require('exit-hook');
+const fs = require('fs');
 
-const text = 'Dist files have the correct js syntax';
+const cjsFields = ['bin', 'main', 'browser'];
+const esFields = ['es2015', 'module'];
+const getFieldFiles = function(cwd, pkg, fields) {
```

Comment:
es check any files in `package.json`, and files in `./es`, `./cjs` `./dist/*.js`  and `./dist/lang/*.js`